lib/ibmdb.js

```diff
@@ -94,6 +94,74 @@ function IBMDB(name, settings) {
 
 util.inherits(IBMDB, SQLConnector);
 
+  IBMDB.prototype.isActual = function(models, cb) {
+    var self = this;
+
+    if ((!cb) && (typeof models === 'function')) {
+      cb = models;
+      models = undefined;
+    }
+
+    // First argument is a model name
+    if (typeof models === 'string') {
+      models = [models];
+    }
+
+    models = models || Object.keys(this._models);
+
+    // var changes = [];
+    async.each(models, function(model, done) {
+      self.getTableStatus(model, function(err, fields, indexes) {
+        if (err) {
+          return done(Error(err));
+        }
+        // TODO: VALIDATE fields/indexes against model definition
```

Comment:
We can implement empty functions here if details are not provided.
So we can remove `isActual` here, use the one in base connector, and add two empty functions `getAddModifyColumns` and `getColumnsToDrop`